frontend/components/NotificationComponent/styles.js

```diff
@@ -0,0 +1,31 @@
+import {StyleSheet} from "react-native";
+
+const styles = StyleSheet.create({
+  view: {
+    flexDirection: 'row',
+    marginTop: 12
+  },
+
+  notification_avatar: {
+    marginHorizontal: 12
+  },
+
+  button: {
+    width: 100,
+    marginEnd: 40,
+    marginVertical: 12
+  },
+
+  button_positive: {
+  },
+
+  button_negative: {
+    backgroundColor: 'grey'
```

Comment:
It is a minor thing but I'd suggest using hexcode or rgb to specify color unless we're dealing with just white or black. You will find a better shade of grey that way. But like I said, this is a very minor issue